tools/semgrep.yml

```diff
@@ -120,3 +120,9 @@ rules:
       - pattern: django.utils.translation.ugettext(...) % ...
     severity: ERROR
     message: "Prefer f-strings or .format for string formatting"
+
+  - id: eval
+    languages: [python]
+    pattern: eval
+    severity: ERROR
+    message: "Do not use eval under any circumstances, no matter how much fun you think it would be"
```

Comment:
We probably should recommend using `json.loads()` instead for places this might come up, to provide a solution?

Reply:
This is unlikely to be a common problem, and suggests deeper issues when it does come up, but sure, I guess.

Reply:
Yeah, it's more a style thing for how we want to write the error messages here (which end up being modeled for others) than a comment on this specific case.